.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,27 @@
+# Dependency Review Action
+#
+# This Action will scan dependency manifest files that change as part of a Pull Request,
+# surfacing known-vulnerable versions of the packages declared or updated in the PR.
+# Once installed, if the workflow run is marked as required,
+# PRs introducing known-vulnerable packages will be blocked from merging.
+#
+# Source repository: https://github.com/actions/dependency-review-action
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
```

Comment:
What is a Harden Runner? Could you add a link to docs here?

Reply:
Sure! Harden Runner acts as a firewall for Github Actions. Currently, it collects all HTTP requests made within the job. The purpose is to review the reports and create specific rules to allow or block requests to unauthorized domains. You can find more information [here](https://github.com/step-security/harden-runner).

Should I add a proper documentation for this?